### PR TITLE
[TO BE REVERTED] Fix Maven IT Failures because of 'no plugins installed'

### PR DESCRIPTION
**Pull request by gregory-paidis-sonarsource**
Fixes https://sonarsource.slack.com/archives/CJ8L44WRF/p1704444673166759

### CODE REVIEW

File: its/src/test/java/com/sonar/it/shared/TestUtils.java [hunk 1]
```diff
@@ -137,7 +137,9 @@ public static ScannerForMSBuild createEndStep(Path projectDir) {
   }
 
   public static ScannerForMSBuild createBeginStep(String projectName, Path projectDir) {
-    return createBeginStep(projectName, projectDir, "");
+    return createBeginStep(projectName, projectDir, "")
+      // FIXME: Remove the next line after this issue is resolved: https://sonarsource.slack.com/archives/CJ8L44WRF/p1704444673166759
```
Comment (andrei-epure-sonarsource) [inline its/src/test/java/com/sonar/it/shared/TestUtils.java:141]: I wonder why this doesn't break the QG...

**Review by andrei-epure-sonarsource — APPROVED** [review]

File: its/src/test/java/com/sonar/it/shared/TestUtils.java (hunk 1 above)
Comment (gregory-paidis-sonarsource) [reply its/src/test/java/com/sonar/it/shared/TestUtils.java:141]: Good point, I left the comment specifically to have the code smell and not forget about it 🤔 
Maybe we are not analyzing the ITs?
I'll take a look a bit later.

Comment (andrei-epure-sonarsource) [reply its/src/test/java/com/sonar/it/shared/TestUtils.java:141]: Yes. And nitpick (not important enough for another CI iteration): we use ToDo for these things.

We use FIXME just during coding for stuff we want to fix before merging a PR.

This is also how the two rules are describing FIXME and ToDo

S1134 - bug you want to fix
[S1135](https://sonarsource.github.io/rspec/#/rspec/S1135) - leftover

Comment (andrei-epure-sonarsource) [reply its/src/test/java/com/sonar/it/shared/TestUtils.java:141]: indeed we only raise on TEST code issues which are only TEST scope
This is both MAIN and TEST so it won't be raised.

**Pull request merged**